src/ctia/properties.clj

```diff
@@ -50,7 +50,11 @@
    (s/optional-key "ctia.producer.es.host") s/Str
    (s/optional-key "ctia.producer.es.port") s/Int
    (s/optional-key "ctia.producer.es.clustername") s/Str
-   (s/optional-key "ctia.producer.es.indexname") s/Str})
+   (s/optional-key "ctia.producer.es.indexname") s/Str
+   (s/optional-key "ctia.producer.es.slicing.strategy")
+   (s/enum :filtered-alias :aliased-index)
+   (s/optional-key "ctia.producer.es.slicing.granularity")
+   (s/enum :minute :hour :day :week :month :year)})
```

Comment:
Having keys and values on separate lines like this is hard to parse unless you add spaces in between.